src/Lecture1.hs

```diff
@@ -123,4 +129,7 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: (Show a, Ord a, Foldable t) => a -> t a -> [Char]
+lowerAndGreater n list = show n ++ " is greater than " ++ show lowerCount ++ " elements and lower than " ++ show higherCount ++ " elements"
+                          where countLowerHigher = foldl (\[l,h] x -> if x < n then [l + 1, h] else if x > n then [l, h + 1] else [l, h]) [0, 0]
+                              [lowerCount, higherCount] = countLowerHigher list
```

Comment:
You can use a tuple `(l, h)` and `(0, 0)` instead of a list. This way, you can pattern-match on values without any warnings 🙂